### PR TITLE
Causes the emergency shuttle to play sounds upon launch and dock

### PR DESCRIPTION
**Pull request by ghost**
@Cuboos has generously created sounds for this project, and they are now used for this PR.

Video demo here: https://dl.dropboxusercontent.com/u/116194515/shuttle_sounds.mp4

### CODE REVIEW

**Comment by Iamgoofball** [comment]
@Cuboos

**Comment by GunHog** [comment]
Have you been reading my dreams, Xhuis!?

**Comment by MMMiracles** [comment]
I tried using a sound effect from TF2 in place for the guitar smash and I got told no, so just forewarning.

**Comment by KorPhaeron** [comment]
Yeah but Tiy stole from all of us so it's only fair

**Comment by Cuboos** [comment]
You uh... want me to make you something a bit more unique?

**Comment by Xhuis** [comment]
@Cuboos If you're up to it, I'd appreciate a contribution, but it isn't necessary.

**Comment by Cuboos** [comment]
THEN WHY AM I HERE?

**Comment by Cuboos** [comment]
"Should we or should we not ask Cuboos to do the JOB HE VOLUNTEERED TO DO"

**Comment by Iamgoofball** [comment]
@KorPhaeron 
all of our sounds are public domain ones we lifted, what did he steal?

**Comment by KorPhaeron** [comment]
~20 million dollars

**Comment by WJohn** [comment]
:+1: x :100: 
These sound amazing.

**Comment by Cuboos** [comment]
https://dl.dropboxusercontent.com/u/9654207/Shuttle_Start.ogg
https://dl.dropboxusercontent.com/u/9654207/ShuttleFly_Amb.ogg
https://dl.dropboxusercontent.com/u/9654207/ShuttleDock.ogg

I have the project files saved, so let me know if you want me to work more on anything.

**Comment by RemieRichards** [comment]
Until proper sounds are added
(like Cuboos's)
this can't be merged.

**Comment by Jordie0608** [comment]
@Cuboos The ambient ogg is 166KB, it could stand to be a much shorter loop.
The end of Start and Dock could probably taper off a bit quicker/less to cut down on their sizes as well (81KB and 55KB respectively) but not as much of an issue.

**Comment by Xhuis** [comment]
The shuttle sounds made by @Cuboos have been added to the PR and used in the game code. The video demo in the OP is also updated.

The "ambience loop" has been shortened to around 5 seconds and made to fade out.

**Comment by Cuboos** [comment]
I wish Byond wasn't shit at looping sounds...

**Comment by Xhuis** [comment]
This is merge-ready

**Comment by Dorsisdwarf** [comment]
Why is there a ~3 second pause between 0:00 and shuttle launch in the video?

**Comment by Xhuis** [comment]
When the timer reaches 0, the sound plays and the launch is timed up with it.

**Comment by Dorsisdwarf** [comment]
I don't like that this effectively adds time to the countdown. Can't you make the shuttle spinning up sound happen *before* the timer ends?

**Comment by Xhuis** [comment]
I tried with no luck.

File: code/controllers/subsystem/shuttles/emergency.dm [hunk 1]
```diff
@@ -105,11 +105,14 @@
 				priority_announce("Hostile enviroment detected. Departure has been postponed indefinitely pending conflict resolution.", null, 'sound/misc/notice1.ogg', "Priority")
 				mode = SHUTTLE_STRANDED
 			if(time_left <= 0 && !SSshuttle.emergencyNoEscape)
```
Comment (Dorsisdwarf) [inline code/controllers/subsystem/shuttles/emergency.dm]: Can't you just add 

if(time_left <=5 && !SSshuttle.emergencyNoEscape)
     play_cool_sound

File: code/controllers/subsystem/shuttles/emergency.dm [hunk 2]
```diff
@@ -105,11 +105,14 @@
...
+				world << 'sound/effects/hyperspace_begin.ogg'
+				sleep(55) //I TRIED PUTTING A SPAWN() HERE AND IT MADE THE SHUTTLE EXPLODE. DON'T TOUCH THIS. Also it's 55 so it times up with the launch.
 				//move each escape pod to its corresponding transit dock
 				for(var/obj/docking_port/mobile/pod/M in SSshuttle.mobile)
 					if(M.z == ZLEVEL_STATION) //Will not launch from the mine/planet
 						M.enterTransit()
 				//now move the actual emergency shuttle to its transit dock
+				world << 'sound/effects/hyperspace_progress.ogg'
```
Comment (Dorsisdwarf) [inline code/controllers/subsystem/shuttles/emergency.dm]: Actually I didn't notice this but the people on the station shouldn't be hearing all the stages, that makes no sense

Comment (Xhuis) [inline code/controllers/subsystem/shuttles/emergency.dm]: I couldn't find a better way.

File: code/controllers/subsystem/shuttles/emergency.dm (hunk 1 above)
Comment (Xhuis) [inline code/controllers/subsystem/shuttles/emergency.dm]: I tried.

File: code/controllers/subsystem/shuttles/emergency.dm (hunk 2 above)
Comment (RemieRichards) [inline code/controllers/subsystem/shuttles/emergency.dm]: I think you can send a sound to an area, and it'll pass it on to mobs inside.

Comment (Iamgoofball) [inline code/controllers/subsystem/shuttles/emergency.dm]: Anyone NEAR should hear it.

Why not make it a play_sound off the engine bits?

Comment (Xhuis) [inline code/controllers/subsystem/shuttles/emergency.dm]: That would make the sound only audible to a few people. In the interest of gameplay, it's being played to the whole shuttle at least.

**Comment by Xhuis** [comment]
I changed the way the sound works. It now plays to all mobs in the emergency shuttle's area.

**Comment by Xhuis** [comment]
Merge-ready @Razharas

**Pull request merged**